tools/file_packager.py

```diff
@@ -191,17 +196,12 @@
 if not has_preloaded or jsoutput == None:
   assert not separate_metadata, 'cannot separate-metadata without both --preloaded files and a specified --js-output'
 
-ret = '''
-var Module;
-'''
-if not no_closure:
-  ret += '''
-if (typeof Module === 'undefined') Module = eval('(function() { try { return Module || {} } catch(e) { return {} } })()');
-'''
-else:
-  ret += '''
-if (typeof Module === 'undefined') Module = {};
-'''
+ret = ''
```

Comment:
nice, this looks perfect now

Reply:
Yeah, I'm actually curious if `var` is still needed there. By protecting it with `if not from_emcc` there should be no more conflict with `emcc.py`, so I've restored `var` in separate branch and will merge it here once Travis CI finishes and there are no new issues (https://travis-ci.org/nazar-pc/emscripten/builds/312557443).

Reply:
Yeah, I think this is more correct with the `var`, as you just did it. It's not a big deal, since this is code that runs in another script tag, so it assigns to the global scope anyhow, but it's cleaner to have the `var`.